### DISCUSSION TITLE
Relè/Relais smart per pulsanti

### DISCUSSION THREAD

**Discussion by blackwiz4rd** [indomus/forum#3127] [GENERALE - Discussioni generiche]
Buongiorno a tutti,
mi chiedevo se avete qualche idea riguardo la possibilità di comandare punti luce tramite la configurazione spesso usata di pulsanti + relè (essendo i collegamenti elettrici più puliti e semplici) utilizzando de prodotti **smart zigbee**, compatibili con zigbee2mqtt.

Ho fatto varie ricerche ma non trovo facilmente dei prodotti compatibili.

La mia idea era di prendere questo ma non sembra abbia la modalità ad impulsi (collegamento 2 way):
4 CH 10A/16A Tuya DIY Zigbee Smart Switch 2-Way Control Light Relay

Un device compatibile è sicuramente il: https://www.zigbee2mqtt.io/devices/064888.html, tuttavia è molto costoso.

Non capisco onestamente se questo device sarebbe compatibile perchè lo `switch_type` è poco chiaro cosa significhi:
https://www.zigbee2mqtt.io/devices/TS0012_switch_module.html

E.s. supponiamo di dover sostituire un relais (finder serie 26) in un impianto esistente.

Un saluto e grazie per l'aiuto,
Luca.

**Comment by blackwiz4rd** [comment]
Ciao a tutti,
mi rispondo da solo. Una possibile soluzione che ho trovato per comandare punti luce smart con pulsanti è attraverso questi relais:
https://www.zigbee2mqtt.io/supported-devices/#e=switch_type

settando lo `switch_type` a temporary. Per comodità sto prendendo quelli con 4 canali (come il SWTZ27). Vi farò sapere se il funzionamento è confermato.

Saluti!

**Reply by 50494554524F** [reply]
arrivo tardi, non li ho ancora comprati... su aliexpress i tuya te li tirano dietro ormai
